apps/deploy-web/src/components/alerts/ContactPointEditContainer/ContactPointEditContainer.tsx

```diff
@@ -0,0 +1,80 @@
+"use client";
+
+import type { FC, ReactNode } from "react";
+import React from "react";
+import { useCallback } from "react";
+import type { components } from "@akashnetwork/react-query-sdk/notifications";
+
+import { useServices } from "@src/context/ServicesProvider";
+import { useNotificator } from "@src/hooks/useNotificator";
+import { useWhen } from "@src/hooks/useWhen";
+
+type ContactPointPatchInput = components["schemas"]["ContactPointPatchInput"]["data"];
+export type ContainerPatchInput = Pick<ContactPointPatchInput, "name"> & {
+  emails: Required<ContactPointPatchInput>["config"]["addresses"];
+};
+
+export type ChildrenProps = {
+  values: Required<ContainerPatchInput>;
+  onEdit: (input: ContainerPatchInput) => void;
+  isLoading: boolean;
+};
+
+type ContactPointEditContainerProps = {
+  id: string;
+  children: (props: ChildrenProps) => ReactNode;
+  onEdit: () => void;
+};
+
+export const ContactPointEditContainer: FC<ContactPointEditContainerProps> = ({ id, children, onEdit }) => {
+  const { notificationsApi } = useServices();
+  const mutation = notificationsApi.v1.patchContactPoint.useMutation({
+    path: {
+      id
+    }
+  });
+  const query = notificationsApi.v1.getContactPoint.useQuery({
+    path: {
+      id
+    }
+  });
+  const notificator = useNotificator();
+
+  const edit = useCallback(
```

Comment:
non-blocking. If you do this, args type can be removed
```suggestion
  const edit: ContactPointEditContainerProps['onEdit] = useCallback(
```

Reply:
Done